swagger.yaml

```
@@ -9,37 +11,34 @@ servers:
   - url: TODO
 
 paths:
-  /reports:
-    get:
-      summary: Restituisce la lista delle segnalazioni.
-      description: Restituisce la lista delle segnalazioni.
-      parameters:
-        - in: query
-          name: offset
-          schema:
-            type: integer
-          description: Numero di segnalazioni da saltare prima di restituire i risultati.
-        - in: query
-          name: limit
-          schema:
-            type: integer
-          description: Numero massimo di segnalazioni da restituire.
-      responses:
-        "200":
-          description: OK
-          content:
-            application/json:
-              schema:
-                type: array
-                items:
-                  $ref: "#/components/schemas/Report"
-        "500":
-          description: Errore interno del server.
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/Error"
-
+  /events:
+    summary: Restituisce la lista degli Eventi.
+    description: Restituisce la lista degli Eventi.
+    parameters:
+      - in: query
+        name: offset
+        schema:
+        type: integer
+        description: Numero massimo di eventi da restituire.
+        name: limit
```

Devi aggiungere in query perchè sono due parametri diversi (ognuno lo si riconosce con -)
```suggestion
      - in: query
        name: limit
```